tests/suites/test_suite_pk.function

```diff
@@ -462,6 +522,10 @@ void pk_utils(int type, int parameter, int bitlen, int len, char *name)
 {
     mbedtls_pk_context pk;
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    PSA_INIT();
+#endif
```

Comment:
Cosmetics: same as previous PR.

(Same note about backwards compat as well: this raises suspicion but is actually OK.)